templates/registration/login.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form method="post" action="{% url "login" %}">
```

Comment:
Ensure that the URL name 'login' matches the URL configuration in your Django project. If you are using Django's built-in authentication views, the URL name should be 'login' as configured in `urls.py`.